auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+"""
+سرویس مدیریت توکن‌ها و احراز هویت
+Token Management and Authentication Service
+"""
+
+from django.contrib.auth import get_user_model
+from django.utils import timezone
+from django.conf import settings
+from datetime import timedelta
+from typing import Tuple, Optional, Dict
```

Ruff (0.12.2)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)